solidity/contracts/test/TestSendReceiver.sol

```diff
@@ -47,10 +49,15 @@ contract TestSendReceiver is IMessageRecipient {
         );
     }
 
-    function handle(uint32, bytes32, bytes calldata) external payable override {
+    function handle(
```

Comment:
can we actually keep the probabilistic reverting as well, and have this just be a special case?

the idea is that we want e2e coverage in the event that a message isn't immediately deliverable but is later